------------------------------------------------------------------------
r1698931 | lgo | 2010-06-02 12:57:39 +0200 (Wed, 02 Jun 2010) | 1 line
Changed paths:
   M /serf/branches/0.6.x
   M /serf/branches/0.6.x/outgoing.c

Merged r1372 from trunk to fix build problem.
------------------------------------------------------------------------
Index: outgoing.c
===================================================================
--- outgoing.c	(revision 1698930)
+++ outgoing.c	(revision 1698931)
@@ -480,7 +480,7 @@ static apr_status_t socket_writev(serf_connection_
                             sizeof(struct iovec) * (conn->vec_len - i));
                     conn->vec_len -= i;
                 }
-                ((char *) conn->vec[0].iov_base) += conn->vec[0].iov_len - (len - written);
+                conn->vec[0].iov_base = (char *)conn->vec[0].iov_base + (conn->vec[0].iov_len - (len - written));
                 conn->vec[0].iov_len = len - written;
                 break;
             }

Property changes on: outgoing.c
___________________________________________________________________
Modified: svn:mergeinfo
## -0,0 +0,1 ##
   Merged /serf/trunk/outgoing.c:r1698909
Index: .
===================================================================
--- .	(revision 1698930)
+++ .	(revision 1698931)

Property changes on: .
___________________________________________________________________
Modified: svn:mergeinfo
## -0,0 +0,1 ##
   Merged /serf/trunk:r1698909
